### PR TITLE
Demographic upload

### PR DESCRIPTION
**Pull request by Erin-Mounts**

### CODE REVIEW

File: BridgeAppSDK/SBAProfileItem.swift [hunk 1]
```diff
@@ -827,10 +838,9 @@ open class SBAClientDataProfileItem: SBAProfileItemBase {
         didSet {
             // get all the SBAClientDataProfileItem instances from SBAProfileManager
             guard scheduledActivities != nil && scheduledActivities!.count > 0,
-                    let clientDataItems: [SBAClientDataProfileItem] = SBAProfileManager.shared?.profileItems().values.mapAndFilter({ (profileItem) -> SBAClientDataProfileItem? in
-                    return profileItem as? SBAClientDataProfileItem
-                }) else { return }
+                    let clientDataItems: [SBAClientDataProfileItem] = SBAProfileManager.shared?.profileItems().values.mapAndFilter({ return $0 as? SBAClientDataProfileItem }) else { return }
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBAProfileItem.swift]: Nit: Getting a bit long. Can you put the `else return` on a new line? Thanks

File: BridgeAppSDK/SBAProfileManager.swift [hunk 2]
```diff
@@ -143,6 +149,9 @@ open class SBAProfileManager: SBADataObject, SBAProfileManagerProtocol {
         return allItems
     }()
     
+    private dynamic var demographicSchemaIdentifier: String?
+    private dynamic var demographicArchiveFilename: String?
```
Comment (syoung-smallwisdom) [inline BridgeAppSDK/SBAProfileManager.swift:153]: I'm not seeing where these are added to the list of keys to parse out of the dictionary. How are they set?

**Review by syoung-smallwisdom — APPROVED** [review]
One nit and a question. Otherwise, looks good

File: BridgeAppSDK/SBAProfileManager.swift (hunk 2 above)
Comment (Erin-Mounts) [reply BridgeAppSDK/SBAProfileManager.swift:153]: whoops

File: BridgeAppSDK/SBAProfileItem.swift (hunk 1 above)
Comment (Erin-Mounts) [reply BridgeAppSDK/SBAProfileItem.swift]: of course

**Pull request merged**